index.html

```diff
@@ -0,0 +1,211 @@
+            <div class="icon">
+              <i class="uil uil-tv-retro"></i>
+              <h3>Excursion</h3>
+            </div>
+            <p>
+              Explore the vast Historic collections at the national Museum, Walk
+              through a range of art works at the Nike Art Garlary
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-comment-alt-notes"></i>
+              <h3>Workshop</h3>
+            </div>
+            <p>
+              The mandatory continuing professional development program that
+              will be hosted in this year’s conference is focused on Vaccination
+              Training for Pharmacists.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-presentation-edit"></i>
+              <h3>Novelty Match</h3>
+            </div>
+            <p>
+              There will be a Novelty Match between the different state arm of
+              the association.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-palette"></i>
+              <h3>Group Dynamics</h3>
+            </div>
+            <p>
+              Every state arm of the association will be required to exhit or
+              showcase their progress and activites within the past one year
+            </p>
+          </div>
+          <div class="join">
+            <a href="#">Join ACPN 2023.</a>
+          </div>
+        </div>
+        <p class="show">SEE THE WHOLE PROGRAM</p>
+      </div>
+    </section>
+
+    <!------------------------------PROGRAMS  END--------------------------------->
+
+    <!------------------------------SPEAKERS  STARTS------------------------------>
+    <section class="speakers">
+      <div class="holder">
+        <h2>Featured Speakers</h2>
+        <hr />
+        <div class="guest-speakers"></div>
+        <!-- <div class="speakers-detail">
+          <img src="" alt="" />
+          <div class="details">
+            <h4>Folashade Lawal</h4>
+            <p class="title">Pharmacist</p>
+            <p class="position">Head Of Pharmacy</p>
+          </div>
+        </div> -->
+      </div>
+    </section>
+    <!------------------------------SPEAKERS  END--------------------------------->
+
+    <!------------------------------PARTNER  START-------------------------------->
+    <section class="partner support" id="support">
+      <div class="holder">
+        <h4>Partners</h4>
+        <hr />
+        <div class="company-logo home-logo">
+          <p class="sponsor1">ASON</p>
+          <img src="images/Alpha-Logo.jpeg" alt="" />
+          <img src="images/May-Baker-Logo.png" alt="" />
+          <img src="images/Oakleaf-Logo.jpeg" alt="" />
+          <img src="images/sanofi.jpg" alt="" />
+          <img src="images/Bydow-Logo.png" alt="" />
+        </div>
```

Comment:
- Like I mentioned for the about page, wrapping this section in a `ul` page and the individual images in a `li` page will help improve the semantics of your page. Please fix this 👍 
- Also, to help improve the accessibility of your site, I suggest including `alt` text to all your images. This would help improve your site when users use screen readers
- Also, please improve the margins, spacings and borders for this section. You can consider using the `flex-wrap:wrap` property in CSS for this section
Your project:
![rharhr](https://user-images.githubusercontent.com/73607512/234033802-5b9fcea7-2822-41ca-8365-728f490d2b16.PNG)